components/HackKnightChallenge/KnightChallenge.tsx

```diff
@@ -0,0 +1,317 @@
+                                            alliances
+                                        </span>
+                                        ) together make an undirected graph with
+                                        multiple connected components, each
+                                        representing a wizarding tribe.
+                                    </p>
+                                    Receive these inputs by making a GET request to
+                                    this endpoint.
+                                    <p
+                                        className={`${styles.colouredText} ${styles.marginTop} ${styles.text}`}
+                                    >{`GET https://artemis.hackillinois.org/challenge`}</p>
+                                </div>
+                            </li>
+                            <li className={styles.text}>
+                                <div className={styles.compact}>
+                                    <b className={styles.text}>
+                                        Problem Statement
+                                    </b>
+                                    <p className={styles.text}>
+                                        The town King has tasked you with finding the
+                                        greatest goodness value among all
+                                        wizarding tribes. In other words, find the goodness values of each tribe and return the greatest of those goodness values. If suceeded, the King will handle the rest! 
+                                    </p>
+                                </div>
+                            </li>
+                            <li className={styles.text}>
+                            <div className={styles.compact}>
+                                    <b className={styles.text}>
+                                        Submitting Your Solution
+                                    </b>
+                                    <p className={styles.text}>
+                                    Make a POST request to
+                                    this endpoint and include your
+                                    {" "}<span className={styles.colouredText}>
+                                            max_goodness
+                                        </span>{" "}
+                                    in the request body as shown below.
+                                    </p>
+                                    <p
+                                        className={`${styles.colouredText} ${styles.marginTop} ${styles.text}`}
+                                    >{`POST https://artemis.hackillinois.org/challenge`}</p>
+                                    <p
+                                        className={`${styles.colouredText} ${styles.marginTop} ${styles.text}`}
+                                    >{`{"max_goodness": token_here}`}</p>
```

Comment:
this line should say `{"max_goodness": integer_value_here}` (not token - my bad)